api.swagger.yaml

```diff
@@ -1845,6 +1845,51 @@ paths:
           description: The UUID or request body JSON was malformed.
         '404':
           description: No manual page with that UUID was found.
+  '/manuals/{manual_page_id}/pages':
+    get:
+      description: Get all pages in a manual or manual page.
+      tags:
+        - manuals
+      parameters:
+        - name: latest
+          description: Whether to return the latest unpublished revision of each page.
+          in: query
+          schema:
+            type: boolean
+            default: false
+        - name: cursor
+          description: The paging cursor.
+          in: query
+          schema:
+            type: string
+            format: uuid
+            example: '01234567-89ab-cdef-1234-56789abcdef0'
+      security:
+        - BearerAuth: []
+      responses:
+        '200':
+          description: Returns a list of manual pages.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
```

Comment:
If you're using a cursor you'll want to include it in the response

Reply:
I was thinking of just using the UUID string of the last item in the result set as the cursor. What would you suggest for the cursor value instead?

Reply:
It can be that, that's no problem.

The point is rather that the API definition should not specify what the cursor is. It should just return a value with the meaning "pass this back unchanged, to get the next item(s)". So if we decide later that it's better to implement it as a simple integer offset, or if we go in the other direction and want to use a JWT or something, we can do that without changing the interface.

However that means we have to explicitly return what the cursor value is - even if it happens to be identical to something else that already appears in the response.

Reply:
Makes sense; done.